### PR TITLE
chore: enhance doc-changelog migration guide

### PR DESCRIPTION
**Pull request by klmcadams**
Updated the doc-changelog migration guide to clarify setting up the release notes in the `CHANGELOG.md` file versus the documentation (`changelog.rst`). Although a couple of the steps are repeated in the "release notes in your documentation" and "release notes in `CHANGELOG.md`" sections, I think it's clearer. 

Previously, you would have to set up everything for the `CHANGELOG.md` configuration, and then change most of what you had set up if you want to include it in your documentation instead.

### CODE REVIEW

File: doc/source/migrations/docs-changelog-setup.rst [hunk 1]
```diff
@@ -3,25 +3,154 @@
...
+    {% endfor %}
+
+    {% endfor %}
+    {% else %}
+    No significant changes.
+
+
+    {% endif %}
+
+|
```
Comment (RobPasMue) [inline doc/source/migrations/docs-changelog-setup.rst:72]:
```suggestion
```

File: doc/source/migrations/docs-changelog-setup.rst [hunk 2]
```diff
@@ -48,9 +177,28 @@ Also, replace ``ansys.<product>.<library>`` with the name under ``tool.flit.modu
...
+    .. code:: md
+
+        This project uses [towncrier](https://towncrier.readthedocs.io/). Changes for the upcoming release can be found in
+        [changelog.rst](doc/source/changelog.rst).
+
 |
 
-2. Create the ``doc/changelog.d`` directory and then within it add a file named ``changelog_template.jinja`` that contains the following lines:
+Reference pull requests for the changes can be found in the `PyAnsys Geometry <https://github.com/ansys/pyansys-geometry/pull/1138>`_ and `PyMechanical <https://github.com/ansys/pymechanical/pull/757/files>`_ repositories.
+The PyAnsys-Geometry pull request includes some other changes, but the changelog implementation is the same as described in this document.
```
Comment (RobPasMue) [inline doc/source/migrations/docs-changelog-setup.rst:196]:
```suggestion
The `PyAnsys Geometry`_ pull request includes some other changes, but the changelog implementation is the same as described in this document.
```

File: doc/source/migrations/docs-changelog-setup.rst [hunk 3]
```diff
@@ -94,116 +242,50 @@ Also, replace ``ansys.<product>.<library>`` with the name under ``tool.flit.modu
...
+    directory = "miscellaneous"
+    name = "Miscellaneous"
+    showcontent = true
 
-        This project uses [towncrier](https://towncrier.readthedocs.io/) and the
-        changes for the upcoming release can be found in
-        this [repository file](doc/changelog.d/changelog.rst).
+|
 
+A reference pull request for these changes can be found in the `PyAnsys-Geometry <https://github.com/ansys/pyansys-geometry/pull/1023/files>`_ repository.
```
Comment (RobPasMue) [inline doc/source/migrations/docs-changelog-setup.rst:287]:
```suggestion
A reference pull request for these changes can be found in the `PyAnsys Geometry #1023 <https://github.com/ansys/pyansys-geometry/pull/1023/files>`_ pull request.
```

**Review by RobPasMue — COMMENTED** [review]
Some small comments. LGTM otherwise!

**Comment by klmcadams** [comment]
@ansys/pyansys-core do you guys prefer having a space between the steps or no space? 

Space:
space

No space:
nospace

**Comment by RobPasMue** [comment]
> @ansys/pyansys-core do you guys prefer having a space between the steps or no space?
> 
> Space: space
> 
> No space: nospace

This is related to: https://github.com/ansys/actions/pull/499#discussion_r1630765632

**Review by SMoraisAnsys — APPROVED** [review]
LGTM !
I'd go with the no space approach. It doesn't look bad without it and avoids unexpected `|` in the file.

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**